packages/core/src/domain/session/sessionCookieStore.spec.ts

```diff
@@ -0,0 +1,199 @@
+import { updateExperimentalFeatures, resetExperimentalFeatures } from '@datadog/browser-core'
+import { stubCookie } from '../../../test/specHelper'
+import {
+  SESSION_COOKIE_NAME,
+  toSessionString,
+  retrieveSession,
+  persistSession,
+  withCookieLockAccess,
+} from './sessionCookieStore'
+import { SessionState } from './sessionStore'
+
+describe('session cookie store', () => {
+  const COOKIE_OPTIONS = {}
+  let initialSession: SessionState
+  let otherSession: SessionState
+  let processSpy: jasmine.Spy<jasmine.Func>
+  let afterSpy: jasmine.Spy<jasmine.Func>
+  let cookie: ReturnType<typeof stubCookie>
+
+  type OnLockCheck = () => { currentState: SessionState; retryState: SessionState }
+
+  function lockScenario({
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏since this function is only used in the last `it`, what about moving it closer to the end of the file?